Refactor-Testing/Refactor-Testing.cpp

```diff
@@ -18,13 +22,69 @@ using namespace Microsoft::VisualStudio::CppUnitTestFramework;
 
 namespace RefactorTesting
 {
-	TEST_CLASS(RefactorTesting)
-	{
-	public:
-		
-		TEST_METHOD(Get_Grid)
-		{
-
-		}
-	};
-}
+    TEST_CLASS(RefactorTesting)
+    {
+    public:
+        
+        NPC autoTestNPC; //NPC that will be used in all the tests
+        TEST_METHOD_INITIALIZE(npcTestSetup)
+        {
+            QuestProgress::Reset(); //Resets the quest progress every time
+        }
+        TEST_METHOD(Get_Grid)
+        {
+
+        }
+
```

Comment:
This won't achieve what you want (assuming you want the test case to pass)
Since you reset the quest to not started - the first Assert will fail.
The objective is typically for all test cases to pass.
Suggestion - change the first Assert to IsFalse(...)